app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    print(command)
+    if command[1] == command[2]:
+        pass
+    elif ".txt" in (command[1] and command[2]):
```

Comment:
Bug: The condition '.txt' in (command[1] and command[2]) is not checking if both files have a '.txt' extension. The 'and' operator returns the second operand if the first is truthy, so you're only checking if '.txt' is in command[2]. You should check both files separately.